evm-ds/src/cps_executor.rs

```diff
@@ -91,11 +94,35 @@ impl<'a> CpsExecutor<'a> {
         feedback: Option<EvmProto::Continuation>,
     ) -> Result<(), evm::ExitError> {
         if let Some(evm_feedback) = feedback {
```

Comment:
Shouldn't we also check here if cps is enabled? Surely, the feedback argument should always be `None` if it isn't, but a bit of defensive approach wouldn't hurt?

Reply:
Yep I can fix that.